neurom/check/morphtree.py

```diff
@@ -173,7 +176,7 @@ def is_inside_cylinder(seg1, seg2):
         return not is_in_the_same_verse(seg1, seg2) and is_seg1_overlapping_with_seg2(seg1, seg2)
 
     # filter out single segment sections
-    for snode in ifilter(lambda snode: snode.points.shape[0] > 2, neurite.iter_sections()):
+    for snode in filter(lambda snode: snode.points.shape[0] > 2, neurite.iter_sections()):
```

Comment:
It does from the:
```
if sys.version_info <= (3, 0):
    from itertools import ifilter as filter
```
at the top.

I think, however, we should consider moving all those checks into a compat.py, and have that use six.py.